README.md

```diff
@@ -1 +1,27 @@
-# Portfolio-Project-
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portfolio-Project-
+
+    This project is a Html and Css Project
+
```

Comment:
Please add a screenshot of your website here. This makes it easier to showcase your project as your README as it's the first thing that the visitors would see before going deeper in your project, and please keep it updated.